sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -643,6 +652,9 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
       npmDevDependencies ++= (npmDevDependencies in Compile).value,
 
+      // Default to deprecated requiresDOM to not break old build.
+      requireJsDomEnv := (requiresDOM in Compile).value,
```

Comment:
In case `requiresDOM` is not defined we should default to `false`:

```suggestion
      requireJsDomEnv := requiresDOM.?.value.getOrElse(false),
```